internal/pkg/manifest/validate.go

```diff
@@ -1459,7 +1457,20 @@ func (q SQSQueueOrBool) validate() error {
 	if q.IsEmpty() {
 		return nil
 	}
-	return q.Advanced.validate()
+	if q.Enabled != nil {
+		return nil
+	}
```

Comment:
Do we need this block?

Reply:
Yes because this block checks for the following manifest config.

```
subscribe:
  topics:
     - name: mytopic
       service: myservice
       queue: true
```

We don't need to validate further if the queue is true

Reply:
I feel like ideally `validate` shouldn't have any assumption like that... (again, ideally) it probably shouldn't make assumptions such as  "if `q.Enabled != nil` then `q.Advanced` must have been empty so we can skip now". It should brainlessly validate as if it knows nothing about logics in other methods such as `UnmarshalYAML`.